src/main/kotlin/apply/config/DatabaseInitializer.kt

```diff
@@ -412,7 +412,15 @@ class DatabaseInitializer(
         val missions = listOf(
             Mission(
                 title = "1주 차 프리코스 - 숫자 야구 게임",
-                description = "https://github.com/woowacourse/java-baseball-precourse",
+                description = """
+                    # 미션 - 숫자 야구 게임
+
+                    ## 🔍 진행 방식
+
+                    - 미션은 **기능 요구 사항, 프로그래밍 요구 사항, 과제 진행 요구 사항** 세 가지로 구성되어 있다.
+                    - 세 개의 요구 사항을 만족하기 위해 노력한다. 특히 기능을 구현하기 전에 기능 목록을 만들고, 기능 단위로 커밋 하는 방식으로 진행한다.
+                    - 기능 요구 사항에 기재되지 않은 내용은 스스로 판단하여 구현한다.
+                """.trimIndent(),
```

Comment:
c: 줄 시작 기준을 파이프 기호로 변경하면 줄바꿈이 훨씬 더 명확해지기 때문에 `trimIndent()`보다는 `trimMargin()`을 사용하는 것이 더 좋아요.

```suggestion
                description = """
                    |# 미션 - 숫자 야구 게임
                    |
                    |## 🔍 진행 방식
                    |
                    |- 미션은 **기능 요구 사항, 프로그래밍 요구 사항, 과제 진행 요구 사항** 세 가지로 구성되어 있다.
                    |- 세 개의 요구 사항을 만족하기 위해 노력한다. 특히 기능을 구현하기 전에 기능 목록을 만들고, 기능 단위로 커밋 하는 방식으로 진행한다.
                    |- 기능 요구 사항에 기재되지 않은 내용은 스스로 판단하여 구현한다.
                """.trimMargin(),
```